server/task_executor_schema_update_ghost_cutover.go

```diff
@@ -16,10 +21,58 @@ func NewSchemaUpdateGhostCutoverTaskExecutor(logger *zap.Logger) TaskExecutor {
 
 // SchemaUpdateGhostCutoverTaskExecutor is the schema update (gh-ost) cutover task executor.
 type SchemaUpdateGhostCutoverTaskExecutor struct {
-	l *zap.Logger //nolint
+	l *zap.Logger
 }
 
 // RunOnce will run SchemaUpdateGhostCutover task once.
 func (exec *SchemaUpdateGhostCutoverTaskExecutor) RunOnce(ctx context.Context, server *Server, task *api.Task) (terminated bool, result *api.TaskRunResultPayload, err error) {
-	return true, &api.TaskRunResultPayload{Detail: "not implemented yet"}, nil
+	defer func() {
+		if r := recover(); r != nil {
+			panicErr, ok := r.(error)
+			if !ok {
+				panicErr = fmt.Errorf("%v", r)
+			}
+			exec.l.Error("SchemaUpdateGhostCutoverTaskExecutor PANIC RECOVER", zap.Error(panicErr), zap.Stack("stack"))
+			terminated = true
+			err = fmt.Errorf("encounter internal error when executing schema update cutover task")
+		}
+	}()
+
+	taskDAGList, err := server.store.FindTaskDAGList(ctx, &api.TaskDAGFind{ToTaskID: task.ID})
+	if err != nil {
+		return true, nil, fmt.Errorf("failed to fetch taskDAG for schema update gh-ost cutover task, id: %v", task.ID)
+	}
+	if len(taskDAGList) != 1 {
```

Comment:
If we only want a single `TaskDAG` from the store, we should implement a `GetTaskDAGByToTaskID` and check the number like this.

https://github.com/bytebase/bytebase/blob/ccfc4ceae787a9953d62d02fbc3eeceb834caad4/store/task.go#L286-L298